griptape/templates/tasks/toolkit_task/system.j2

```diff
@@ -1,39 +1,24 @@
-...repeat Thought/Action/Observation as many times as you need
+Thought: <your step-by-step thought process describing what APIs you need to use>
+Request: {{ api_schema }}
+{{ stop_sequence }} <API output>
+...repeat Thought/Request/Response as many times as you need
 Answer: <final answer>
 
-"Thought", "Action", "Observation", and "Answer" MUST ALWAYS start on a new line. If an Observation contains an error, you MUST ALWAYS try to fix the error with another Thought/Action/Observation. NEVER make up action types. NEVER make up action names. NEVER make up action activities. Actions must ALWAYS be plain JSON objects. NEVER make up facts. Be truthful. ALWAYS be proactive and NEVER ask the user for more information input. You should ALWAYS use an action if you can. Keep going until you have the final answer.
-
-{% if tool_names|length > 0 %}
-Actions of Type "tool"
+"Thought", "Request", "Response", and "Answer" MUST ALWAYS start on a new line. If a Response contains an error, you MUST ALWAYS try to fix the error with another Thought/Request/Response. NEVER make up API requests. API requests must ALWAYS be plain JSON objects that follow the schema. NEVER make up facts. Be truthful. ALWAYS be proactive and NEVER ask the user for more information input. You should ALWAYS use an API if you can. Keep going until you have the final answer.
 
-You can use tools. Tools are a type of Action that can help you answer questions. You have access ONLY to the tools with the following names: [{{ tool_names }}]. You can use multiple tools in a sequence to get the final answer. NEVER make up tool names or tool activities.
+{% if api_names|length > 0 %}
+You have access ONLY to the APIs with the following names: [{{ api_names }}]. You can use multiple APIs in a sequence to get the final answer. NEVER make up API names or API paths.
 
-Tool Descriptions:
-{% for tool in tools %}
-{{ tool }}
-{% endfor %}
-
-{% if memory_names|length > 0 %}
-Actions of Type "memory"
-
-Tools might store their output in memory as artifacts (with `memory_name` and `artifact_namespace`). If tool output is stored in memory, ALWAYS try to retrieve it or pass it to another action. You ONLY have access to memory with the following names: [{{ memory_names }}]. NEVER make up memory names or memory activities.
-
-Memory Descriptions:
-{% for memory in memories %}
-{{ memory }}
+API Descriptions:
+{% for api in apis %}
+{{ api }}
 {% endfor %}
 {% endif %}
-{% endif %}
 
-{% if tool_names|length > 0 and memory_names|length > 0 %}
-If tool output is stored in memory, ALWAYS pass it to another tool OR use memory activities to access it.
+{% if memory_names|length > 0 %}
+Tools might store their output in memory as artifacts (with `memory_name` and `artifact_namespace`). If API response is stored in memory, ALWAYS try to pass it to another API. You ONLY have access to memory with the following names: [{{ memory_names }}]. NEVER make up memory names or artifact namespaces.
```

Comment:
Should we be more explicit and call `memory` `tool memory`? Will that confuse it? I'm just worried that `memory` can mean many things.

Reply:
Removing all references to tools. Should just be "actions" and "memory." Let's see how it performs.